### ISSUE TITLE
Do we explain p, q, r in section 6? (Or before it?)

### ISSUE THREAD

**Issue by flighttestfact** [Society-of-Flight-Test-Engineers/handbook-2013#206]

**Comment by cooknl** [comment]
They are described in section 1, both in the listing of common
abbreviations and in the sign conventions.

There's likely room for consolidation in the next version.

**Comment by rwstanfo** [comment]
There is a lot of inconsistency with the introduction of variables.  Not all variables introduced in the beginning of a section are actually used, and in some cases new variables are introduced later in the section (sometimes without a definition).  I've marked the cases I've found with comments in the source code and/or github issues.

**Comment by cooknl** [comment]
Yeah, this is an area that may exceed the "just tweaking the port as we go" threshold (making tables where there weren't tables before but it cleans up the content that already existed would be in the "tweaking" category, for example).

The H2.0 is looking more and more like a fundamental restructure, including how variables and notation are treated. Do we want to keep each section so independent, for example?
